drivers/Makefile.dep

```diff
@@ -645,13 +645,23 @@ ifneq (,$(filter ws281x,$(USEMODULE)))
     ifneq (,$(filter arch_avr8,$(FEATURES_USED)))
       USEMODULE += ws281x_atmega
     endif
+    # fall back to VT100 terminal output if no other
+    # backend is availiable.
+    ifeq (,$(filter ws281x_%,$(USEMODULE)))
+      USEMODULE += ws281x_vt100
+    endif
   endif
   ifneq (,$(filter ws281x_atmega,$(USEMODULE)))
     FEATURES_REQUIRED += arch_avr8
   endif
   USEMODULE += xtimer
 endif
 
+ifneq (,$(filter ws281x_vt100,$(USEMODULE)))
```

Comment:
This block should be moved to `drivers/Makefile.include`

Reply:
But it looks so out of place there. :open_mouth:

Reply:
The idea of the `Makefile.dep` is to set/resolve the dependencies of a module in the build. `Makefile.include` should contain code for setting build configuration variables. For me, CFLAGS is a build configuration variable, so it should be in drivers/Makefile.include.
git grep gives an opposite result as what I just said but I think the already existing CFLAGS defined in Makefile.dep should be moved in Makefile.include.

Reply:
I think the cleanest option would be to have those defines in a header file. Via `CFLAGS`, those defines are available for every module and compilation unit, even though they are only needed for the `ws281x` module.

I see to options for using a header file for this:

a) A common header, let's say `ws281x_backend.h`, which uses `#ifdef MODULE_WS281X_<BACKEND>` to provide the required defines.
b) A separated `ws281x_backend.h` for each backend located at `drivers/ws281x/include/<BACKEND_NAME>/ws281x_backend.h`. And `Makefile.include` is used to add `drivers/ws281x/include/<BACKEND_NAME>` to `INCLUDES` for the used backend.

Reply:
The simpler the better - I prefer option a).